release-notes/next.yml

```diff
@@ -1,4 +1,8 @@
 releases:
+  42.20250609.1.0:
+    issues:
+      - text: "performance regression on aarch64 starting with 42.20250512.X.0"
```

Comment:
I'm wondering if we should re-title the issue (and subsequently update this text). WDYT?

It looks like the issue was more specific than just all ARM machines.. seemed like it was specific to graviton AWS instances..

Suggestion:

- `Performance regression on aarch64 AWS graviton machines`